.env.example

```diff
@@ -1,4 +1,6 @@
+AI_SERVER_URL="http://localhost:8000"
 DISCORD_BOT_TOKEN="EXAMPLETokenTTTTT.Grz0fK.IUi1IONPcLHZsnyjROTt8lR2fuYCeK4grzRoSQ"
+OPENAI_API_KEY="ee-example8888888888888888" # needed for running litellm
 
 # THIS SHOULD BE ASSIGNED TO THE BOT'S `scopes` WHEN DEPLOYED
 DISCORD_GUILD_ID="99999999999999999999999"
```

Comment:
I would recommend lumping all Discord related envs together, and all LiteLLM/OpenAI/... things together.

```suggestion
# LiteLLM/OpenAI
AI_SERVER_URL="http://localhost:8000"
OPENAI_API_KEY="ee-example8888888888888888" # needed for running litellm

# Discord
DISCORD_BOT_TOKEN="EXAMPLETokenTTTTT.Grz0fK.IUi1IONPcLHZsnyjROTt8lR2fuYCeK4grzRoSQ"
# THIS SHOULD BE ASSIGNED TO THE BOT'S `scopes` WHEN DEPLOYED
DISCORD_GUILD_ID="99999999999999999999999"
```

Reply:
My second question is, if `OPEN_API_KEY` isn't neede for LiteLLM (as we only need a fake key), why don't we just hard code it in the code?

Reply:
@samhwang we still need to assign it to ENV before running LiteLLM

in `src/litellm/README.md`:
```shell
# if you use OpenAI models
export OPENAI_API_KEY=<your_key>
litellm --config llm_assistant/ollama/proxy_config.yaml
```

Reply:
What I mean is you won't need it in the code, esp. if you put an export